fixtures/u2_import_path.py.sem.uast

```diff
@@ -0,0 +1,110 @@
+{ '@type': "Module",
+   '@role': [File, Module],
+   '@pos': { '@type': "uast:Positions",
+   },
+   body: [
+      { '@type': "Assign",
+         '@role': [Assignment, Binary, Expression],
+         '@pos': { '@type': "uast:Positions",
+            start: { '@type': "uast:Position",
+               offset: 0,
+               line: 1,
+               col: 1,
+            },
+         },
+         targets: [
+            { '@type': "uast:Identifier",
+               '@role': [Left],
+               '@pos': { '@type': "uast:Positions",
+                  start: { '@type': "uast:Position",
+                     offset: 0,
+                     line: 1,
+                     col: 1,
+                  },
+                  end: { '@type': "uast:Position",
+                     offset: 0,
+                     line: 1,
+                     col: 1,
+                  },
+               },
+               Name: "a",
+            },
+         ],
+         value: { '@type': "Call",
```

Comment:
We could convert an `__import__()` to `RuntimeImport` as well. Thoughts?

Reply:
Unsure. There are actually tons of different ways to import things in python:

1. Read and eval and assign to a global name in the module table.
2. imp module.
3. importlib
4. ...many others that I don't remember but I've seen in the past, some of those pretty exotic.

But is true that `__import__` is a default function in Python so that case at least could be easy to handle.

Reply:
After some consideration, I've decided that I'll not convert `__import__` in this version:

- The argument could be a string, but also could be a variable, a function call, etc so we don't have a real way to know the `RuntimeImport.path`.

- It will be assigned to a returned value that could be different from the argument.

This would need pretty complex annotations or flow analysis in the native drivers and thus I think is better to leave it to future versions to don't delay this anymore.